chore: add `spotify-launcher` aur install details (#92)
spicetify/spicetify-docs: docs/advanced-usage/installation.md

Before applying Spicetify, you need to gain write permission on Spotify files, by running command:

> <!--This may be outdated as the AUR package for the spotify launcher uses a different install directory-->

Reviewer: It looks like `spotify-launcher` is in the [extra] repo, not AUR. Or are you referring to something else?

Author: Oh, sorry. I was referring to spotify-launcher, must have written the wrong thing. Sorry about that, it is in [extra]

Reviewer 2: Then this comment does not really apply right? Since it does still apply, just for a different installation method.
```suggestion
```

```bash
sudo chmod a+wr /opt/spotify
sudo chmod a+wr /opt/spotify/Apps -R
```

If spotify is installed through the `spotify-launcher` package in the AUR then spotify won't install to `/opt/spotify` and is instead in this folder: `~/.local/share/spotify-launcher/install/usr/share/spotify/`

> This directory will need to be added to the `spotify-path` section of the config <!--I Don't think the write permission commands are needed with the home directory-->

Reviewer: If it installs to the home directory, then you won't need to change any permissions, no.

> If spotify is installed through the `spotify-launcher` package in the AUR then spotify won't install to `/opt/spotify` and is instead in this folder: `~/.local/share/spotify-launcher/install/usr/share/spotify/`
> 
> This directory will need to be added to the `spotify-path` section of the config <!--I Don't think the write permission commands are needed with the home directory-->

Reviewer: ```suggestion
#### Spotify installed via `spotify-launcher` package (Arch Linux)

If Spotify is installed through the `spotify-launcher` package, then Spotify won't install to `/opt/spotify` and is instead in this folder: `~/.local/share/spotify-launcher/install/usr/share/spotify/`

This directory will need to be added to the `spotify-path` section of the config (and you won't need to change any permissions like the AUR method). 
```

Before applying Spicetify, you need to gain write permission on Spotify files, by running command:

<!--This may be outdated as the AUR package for the spotify launcher uses a different install directory-->
```bash
sudo chmod a+wr /opt/spotify
sudo chmod a+wr /opt/spotify/Apps -R
```

> #### Spotify installed via `spotify-launcher` package (Arch Linux)
> 
> If Spotify is installed through the `spotify-launcher` package, then Spotify won't install to `/opt/spotify` and is instead in this folder: `~/.local/share/spotify-launcher/install/usr/share/spotify/`
> 
> This directory will need to be added to the `spotify-path` section of the config (and you won't need to change any permissions like the AUR method). 
> 
> **Note:** Your Spotify client location might be different.

Reviewer 2: ```suggestion
**Note:** Your Spotify client location might be different.

#### Spotify installed via `spotify-launcher` package (Arch Linux)

If Spotify is installed through the `spotify-launcher` package, then Spotify won't install to `/opt/spotify` and is instead in this folder: `~/.local/share/spotify-launcher/install/usr/share/spotify/`

This directory will need to be added to the `spotify-path` section of the config (and you won't need to change any permissions like the AUR method). 
```
